src/AppContainer.jsx

```diff
@@ -82,13 +84,16 @@ const AppContainer = ({ store, lang, history, client }) => {
                     <BanksProvider client={client}>
                       <SelectionProvider>
                         <StoreURLProvider>
-                          <MuiCozyTheme>
-                            <RealTimeQueries doctype={TRIGGER_DOCTYPE} />
-                            <RealTimeQueries doctype={ACCOUNT_DOCTYPE} />
-                            <RealTimeQueries doctype={TRANSACTION_DOCTYPE} />
-                            <RealTimeQueries doctype={GROUP_DOCTYPE} />
-                            <Router history={history} routes={AppRoute()} />
-                          </MuiCozyTheme>
+                          <CozyConfirmDialogProvider>
+                            <MuiCozyTheme>
+                              <RealTimeQueries doctype={TRIGGER_DOCTYPE} />
+                              <RealTimeQueries doctype={ACCOUNT_DOCTYPE} />
+                              <RealTimeQueries doctype={COZY_ACCOUNT_DOCTYPE} />
+                              <RealTimeQueries doctype={TRANSACTION_DOCTYPE} />
+                              <RealTimeQueries doctype={GROUP_DOCTYPE} />
+                              <Router history={history} routes={AppRoute()} />
+                            </MuiCozyTheme>
+                          </CozyConfirmDialogProvider>
                         </StoreURLProvider>
                       </SelectionProvider>
                     </BanksProvider>
```

Comment:
nit: It's easier to read, understand  and maintain if this commit was split into both commits: 
- One for the harvest upgrade and the addition of CozyConfirmDialogProvider 
- One other for the use of cozy_account_doctype realtime.